### PR TITLE
Elasticsearch module related documentation

### PR DESCRIPTION
**Pull request by dpobel**

### CODE REVIEW

File: source/docs/appendices/migration-guides.md [hunk 1]
```diff
@@ -51,6 +51,47 @@ This first feature is RMA (Return Merchandize Authorization). In order to add su
...
+
+```diff
+// .front-commerce.js
+-  modules: [],
++  modules: ["./node_modules/front-commerce/modules/datasource-elasticsearch"],
+   serverModules: [
+     { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
+     { name: "Magento2", path: "server/modules/magento2" },
++    {
++      name: "Magento2 Elasticsearch",
```
Comment (JulienPradet) [inline source/docs/appendices/migration-guides.md:68]: This won't work. No spaces are allowed in the names here.

File: source/docs/appendices/migration-guides.md [hunk 2]
```diff
@@ -51,6 +51,47 @@ This first feature is RMA (Return Merchandize Authorization). In order to add su
...
+
+```diff
+// .front-commerce.js
+-  modules: [],
++  modules: ["./node_modules/front-commerce/modules/datasource-elasticsearch"],
+   serverModules: [
+     { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
+     { name: "Magento1", path: "server/modules/magento1" },
++    {
++      name: "Magento1 Elasticsearch",
```
Comment (JulienPradet) [inline source/docs/appendices/migration-guides.md:84]: Same issue here

**Review by JulienPradet — APPROVED** [review]
Thanks for the docs. :)

File: source/docs/appendices/migration-guides.md [hunk 3]
```diff
@@ -51,6 +51,47 @@ This first feature is RMA (Return Merchandize Authorization). In order to add su
...
+`.front-commerce.js`. For a Magento2 based Front-Commerce setup:
+
+```diff
+// .front-commerce.js
+-  modules: [],
++  modules: ["./node_modules/front-commerce/modules/datasource-elasticsearch"],
+   serverModules: [
+     { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
+     { name: "Magento2", path: "server/modules/magento2" },
++    {
```
Comment (dpobel) [inline source/docs/appendices/migration-guides.md:67]: we've just discovered with Julien that the module ES module needs to be registered before
the Magento2 one. This needs to be documented properly with a warning here (and then
we can fix that in an upcoming release)

**Pull request merged**